catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -329,6 +358,58 @@ private CreateResponse createMetacards(CatalogFacade catalog, List<Metacard> lis
         return catalog.create(createRequest);
     }
 
+    private void createContentItem(Metacard metacard) {
+        if (fileMap == null) {
+            return;
+        }
+
+        List<File> fileList = fileMap.get(metacard.getId());
+
+        if (fileList != null) {
+            List<ContentItem> contentItemList = new ArrayList<>();
+            ContentItem contentItem;
+
+            for (File file : fileList) {
+                ByteSource byteSource = com.google.common.io.Files.asByteSource(file);
+                // The fileName is checked earlier and is guaranteed to follow correct convention
+                String fileName = file.getName()
```

Comment:
Maybe create a pair of utlity methods that create and parse filenames following the naming convention. The javadoc of each method can refer to each other, that way everyone knows that both methods must follow the same convention.